src/connectors/kcrw.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.play';
+
+Connector.artistSelector = '.episode-name';
+
+Connector.trackSelector = '.song-name';
+
+Connector.isPlaying = () => $('.play').hasClass('.active');
```

Comment:
```suggestion
Connector.isPlaying = () => Util.hasElementClass('.play', 'active');
```

We're trying to phase out jQuery so this would be better :)